book/src/01_getting_started/03_test.md

```diff
@@ -0,0 +1,42 @@
+# Test
+
+Functionality in EDR is tested in two way:
```

Comment:
Suggestion: `two ways`. (The original is singular.)

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4262